test/observe_test.dart

```diff
@@ -106,6 +108,7 @@ void _observeTests(createModel(x)) {
 
     subs.add(t.changes.listen((n) {}));
     expect(t.hasObservers, true);
+    Observable.dirtyCheck();
```

Comment:
Sorry I should have linked the bug I created for this https://github.com/dart-lang/observe/issues/81. Basically it seems that the tests packages zones are messing up the dirty checking zone.